src/pages/register-a-beacon/activity.tsx

```diff
@@ -61,12 +74,38 @@ const Activity: FunctionComponent<FormPageProps> = ({
       showCookieBanner={showCookieBanner}
       formErrors={form.errorSummary}
       errorMessages={form.fields.activity.errorMessages}
+      insetText={insetText}
     >
-      <RadioList conditional={true}>{Options}</RadioList>
+      <RadioList conditional={true}>
+        <ActivityOptions
+          environment={environment}
+          purpose={purpose}
+          form={form}
+          listItemName={"activity"}
+        />
+      </RadioList>
     </BeaconsForm>
   );
 };
 
+export const ActivityOptions: FunctionComponent<ActivityOptionsProps> = ({
+  environment,
+  purpose,
+  form,
+  listItemName,
+}: ActivityOptionsProps): JSX.Element => {
+  if (environment === Environment.MARITIME && purpose === Purpose.PLEASURE) {
+    return <MaritimePleasureOptions form={form} listItemName={listItemName} />;
+  } else if (
+    environment === Environment.MARITIME &&
+    purpose === Purpose.COMMERCIAL
+  ) {
+    return (
+      <MaritimeCommercialOptions form={form} listItemName={listItemName} />
+    );
```

Comment:
There probably is a nicer way than this to have different views? 🙈  I could nest if's? As in: 
if (MARITIME) {
     if (PLEASURE) {}
     if (COMMERCIAL){}
}